curvefs/src/tools/curvefs_tool_define.h

```diff
@@ -80,4 +85,22 @@ const char kZoneNum[] = "zonenum";
 }  // namespace mds
 }  // namespace curvefs
 
+namespace curvefs {
+namespace tools {
+
+void SetStringFlagInfo(curve::common::Configuration* conf,
+                       google::CommandLineFlagInfo* info,
+                       const std::string& key, fLS::clstring* flag);
+
+void SetUInt64FlagInfo(curve::common::Configuration* conf,
+                       google::CommandLineFlagInfo* info,
+                       const std::string& key, uint64_t* flag);
```

Comment:
use template, and curve::common::Configuration has `GetValueFatalIfFail` for almost all types